libs/common/database/src/lib/DataManagerBase.ts

```diff
@@ -78,7 +78,10 @@ export class DataManagerBase<
       | ((v: StorageValue) => Partial<StorageValue> | void),
     notify = true
   ): boolean {
-    const old = this.getStorage(key)
+    const old =
```

Comment:
I think this is probably safe enough to be rolledout for all instance of DataManagerBase, instead of creating an one-off here.

Reply:
iirc it was causing errors when I enabled it for all instances